### DISCUSSION TITLE
Follow-Up: Pro License Activation Issue for Herd - Urgent Assistance Needed

### DISCUSSION THREAD

**Discussion by mauriciodulce** [beyondcode/herd-community#929] [Q&A] [answered]
I am writing to follow up on the issue I reported regarding the activation of my Herd Pro license. As per your previous instructions, I am reaching out via email to provide further details.

**Platform:** macOS  
**Operating system version:** Version 14.5 (23F79)  
**System architecture:** ARM64 (M1, M2, etc)  
**Herd Version:** 1.9.1 build 31  
**PHP Version:** All versions  

**Bug description:**  
I am unable to activate the Pro license for Herd. Each time I attempt to activate the license, an error message appears indicating that something went wrong. I have tried reinstalling Herd multiple times, both from the official website and using Homebrew. I have also changed the license token and ensured it is copied correctly, but the issue persists. I have sent logs and videos as requested. Additionally, I don’t understand why it takes so long for support to respond.

**Steps to reproduce:**
1. Download and install Herd from the official website or using Homebrew.
2. Obtain a valid Pro license token.
3. Open Herd and navigate to the license activation section.
4. Enter the Pro license token and click on the "Activate License" button.
5. Observe the error message that appears after attempting activation.

**Additional Information:**  
I have been dealing with this issue for two weeks now and have already sent multiple emails without receiving a resolution. Diana and Marcel have replied, but the problem remains unresolved. I am still unable to activate the license that I paid for and have not received a satisfactory solution. I have even sent them videos and logs as requested. Additionally, I am frustrated by the delay in their responses.

I would greatly appreciate your urgent assistance in resolving this matter. Please let me know if any additional information is required.

Thank you for your attention to this issue.

**Comment by sschlein** [comment, marked as answer]
I am closing this because this has been resolved with a refund. We haven't been able to find the cause for that and it hasn't happened to anyone else – something on the device works differently and prevents Herd from activating the license.
